src/component/layout.less

```diff
@@ -13,57 +25,118 @@ ul {
+@Poppins: "Poppins", Arial, Helvetica, sans-serif;
+
 .login-page {
   width: 100%;
   height: 100vh;
+  padding: 0;
   display: flex;
-  align-items: center;
-  justify-content: center;
-  background: linear-gradient(to bottom right, #220c54, #300193);
-  .background {
-    width: 100%;
-    height: 100vh;
+  flex-direction: column;
+  justify-content: flex-start;
+  background-image: url("../assets/login_page_background_image.png");
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  header {
     display: flex;
+    flex-direction: column;
     align-items: center;
-    justify-content: space-between;
-    .responsive(@lg, {
-      background-image: url("../assets/endtoend-encryption-img.webp");
-      background-repeat: no-repeat;
-      background-position: bottom -50% right -10%;
-      background-size: 70%;
-    });
-    .text-content {
-      width: 100%;
-      display: flex;
-      flex-direction: column;
-      align-items: flex-start;
-      gap: 0px;
-      padding: 30px;
-      .responsive(@lg, {
-        width: 50%;
-        padding: 50px;
-      });
+    justify-content: center;
+    padding: 2.5vh 0px;
+    width: 100%;
+    height: 10vh;
+    background: #ffffff10;
+    -webkit-backdrop-filter: blur(50px);
+    backdrop-filter: blur(50px);
+    position: unset; // Bypass the default header style
+    border-bottom: 0.1px solid #ffffff40;
+    img {
+      height: 66%;
     }
+  }
+  .text-content {
+    width: 100%;
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    gap: 5%;
     p {
-      font-size: 2em;
-      font-family: @montserrat;
-      font-weight: 600;
-      line-height: 1.1em;
+      text-align: center;
       color: white;
+      line-height: 1.25;
     }
     p.title {
-      background-image: linear-gradient(to right, #ff8c4a, @primary-deep);
-      background-size: 100%;
-      background-repeat: repeat;
+      font-family: @Poppins;
       font-weight: 700;
-      font-size: 3em;
-      -webkit-background-clip: text;
-      -webkit-text-fill-color: transparent;
-      -moz-background-clip: text;
-      -moz-text-fill-color: transparent;
-    }
-    button {
-      font-size: 1.2em;
+      font-size: 10vh;
+      text-align: center;
+      margin-bottom: unset; // bypass default margin
+      @media only screen and (max-width: @sm) {
+        font-size: 6vh;
+      }
+    }
+    p.subtitle {
+      font-family: @Poppins;
+      font-weight: 600;
+      font-size: 3rem;
+      margin: 0 5%;
+      opacity: 80%;
+      @media only screen and (max-width: @sm) {
+        font-size: 1.5rem;
+      }
+    }
+    .buttons {
+      width: 100%;
+      button {
+        cursor: pointer;
+        display: flex;
+        flex-direction: row;
+        justify-content: center;
+        align-items: center;
+        --border-radius: 44px;
+        border-radius: var(--border-radius);
+        width: 200px;
+        height: 78px;
```

Comment:
please reduce this size again - we said around 150px for the width to get the same proportion as the Figma (and adjust height too of course)

Reply:
![image](https://github.com/Cosmian/interactive-tutorial-ui/assets/19950216/d31621ce-f5e1-490d-a3dd-031089990129)

I ended up chosing 180px && 70px - this is how it looks like on my laptop